src/app/app-routing.module.ts

```diff
@@ -1,7 +1,11 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { CoursesListComponent } from './features/courses/courses-list/courses-list.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: '', redirectTo: 'courses', pathMatch: 'full' },
+  { path: 'courses', component: CoursesListComponent }
```

Comment:
if you have some string and you could move it into 'const' would be better do it.
In this case for example 

const RootRoutes = {
  courses: 'courses'
}; 
and use it instead of string